src/index.js

```diff
@@ -12,9 +12,9 @@ const root = createRoot(container);
 root.render(
   <BrowserRouter>
     <React.StrictMode>
```

Comment:
Ohhh StrictMode.... what do you do??

Reply:
Y'all should dive into that!  It could be a good interview question!  😉